### PR TITLE
feat: A couple more Acs

### PR DESCRIPTION
**Pull request by TomMcL**

### CODE REVIEW

File: protocol/0074-BTCH-batch-market-instructions.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ Overall, building the ability to handle batches of market instructions in a sing
   - **Cancellations**: this is a list (repeated field) of Cancel Order instructions
   - **Amendments**: this is a list (repeated field) of Amend Order instructions
   - **Submissions**: this is a list (repeated field) of Submit Order instructions
-- Additionally the batch may contain a single transaction to change the current margin mode.
+- Additionally the batch may contain a single transaction to change the current margin mode. Unlike cancellations/amendments/submissions, if this fails the rest of the batch will be cancelled.
```
Comment (barnabee) [inline protocol/0074-BTCH-batch-market-instructions.md]: Should this not only apply to other [submissions|all market instructions] for the same market?

File: protocol/0019-MCAL-margin_calculator.md [hunk 2]
```diff
@@ -280,6 +280,8 @@ The margin account should have additional amount `limit price x size x margin fa
 
 - For each market and each party which has either orders or positions on the market, the API provides the maintenance margin levels.  (<a name="0019-MCAL-092" href="#0019-MCAL-092">0019-MCAL-092</a>)
 
+- For each market and each party which has either orders or positions on the market, the API provides the current margin mode and, when in isolated margin mode, leverage setting.  (<a name="0019-MCAL-142" href="#0019-MCAL-142">0019-MCAL-142</a>)
```
Comment (Sohill-Patel) [inline protocol/0019-MCAL-margin_calculator.md]: does this ac apply all of these apis?

- ListMarginLevels
- ObserveMarginLevels
- EstimateMargin

Comment (TomMcL) [reply protocol/0019-MCAL-margin_calculator.md]: `EstimateMargin` is going to be a different one, as that is a standalone calculation, but the other two yes

Comment (Sohill-Patel) [reply protocol/0019-MCAL-margin_calculator.md]: ok great! And what are `leverage settings`?

Comment (TomMcL) [reply protocol/0019-MCAL-margin_calculator.md]: The margin ratio, I could make that clearer :D

**Review by Jiajia-Cui — APPROVED** [review]
LGTM

**Pull request merged**